src/SFA.DAS.LevyTransferMatching.Web/Views/Pledges/Close.cshtml

```diff
@@ -0,0 +1,75 @@
+﻿@model SFA.DAS.LevyTransferMatching.Web.Models.Pledges.CloseViewModel
+
+@{
+    ViewData["Title"] = "Close pledge";
+}
+
+<a href="@Url.Action("Applications", new { EncodedAccountId = Model.EncodedAccountId, EncodedPledgeId = Model.EncodedPledgeId })" class="govuk-back-link">Back</a>
+
+<main class="govuk-main-wrapper" id="main-content" role="main">
+
+    <partial name="_ValidationSummary" />
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+
+            <h1 class="govuk-heading-xl">Close pledge @Model.EncodedPledgeId </h1>
```

Comment:
Whitespace after pledge id removed. One blank line, out of the two, after the above line also removed.
Other blank lines kept for the readability but can be removed if required, so that all of the view file doesn't have any blank lines.